.github/workflows/check-e2e.yml

```diff
@@ -36,10 +36,10 @@ jobs:
         if: steps.playwright-dep-cache.outputs.cache-hit != 'true'
         run: npx playwright install chromium firefox
 
-      - name: Start http-api test server
+      - name: Install binaries
         run: |
           mkdir -p tests/artifacts;
-          ./scripts/run-httpd-with-fixtures --non-interactive --download 2>&1 | tee tests/artifacts/httpd-${{ matrix.browser }}.log &
+          source ./scripts/install-binaries;
```

Comment:
I'm a bit curious as to why this works. It seems to me that there's some unexpected side-effect that makes it work. But I guess it doesn't matter, because we'll pass the path via GITHUB_OUTPUT to the next step as discussed.

Reply:
In the end I wrote a small script to compute the current binaries folder based on the `heartwood-version` file.
So we can just run the `install-binaries` script and then pass the output of `get-binaries-folder` to `$GITHUB_OUTPUT` and export it to PATH in the required step